pkg/didcomm/common/service/destination.go

```diff
@@ -19,6 +28,7 @@ type Destination struct {
 	ServiceEndpoint      string
 	RoutingKeys          []string
 	TransportReturnRoute string
+	RecipientEncKeys     []*composite.PublicKey
```

Comment:
Why isn't `RecipientKeys` deleted?

Reply:
`RecipientKeys` contains remain verification keys and are also used by LegacyPacker

Reply:
I guess in a future PR we will even have RecipientVerKeys as per [this](https://github.com/hyperledger/aries-framework-go/pull/2068#discussion_r465173320) comment.